lib/albacore/task_types/test_runner.rb

```diff
@@ -191,7 +191,7 @@ def handle_directory dll, exe, &block
                 [Paths.normalise_slashes(Pathname.new(File.dirname(dll))), Pathname.new(exe)]
             end
 
-          exe_rel = exe.relative_path_from dir
+          exe_rel = exe.to_s.include?(':') ? exe : exe.relative_path_from(dir)
```

Comment:
Either one, I prefer expand_path, doesn't need to create an object.  Let me know which we want to end up with, I'll update PR.